.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,35 @@
-## What changed?
-
-🦄...
-
-### Definition of Done
-
-- [ ] Dependencies checked (if applicable)
-- [ ] TS types updated (if applicable)
-- [ ] CHANGELOG.md updated (if applicable)
-- [ ] Complete unit testing (if applicable)
-- [ ] Complete screenshot testing (if applicable)
-- [ ] Complete UX review (if applicable)
-- [ ] Documentation design guide updated (if any)
-- [ ] Documentation examples updated (if any)
-- [ ] Documentation API updated (if any)
-- [ ] Add link to design (if any)
-- [ ] No major bugs pending
-- [ ] Complete self code review
-
-### Reviewer checklist
-
-- [ ] Does the code work? Does it perform its intended function and the logic is correct?
-- [ ] Is the code easily understandable?
-- [ ] Verify the design is implemented as per the design requirements
-- [ ] Verify the names used in the programs/methods/functions convey the intent and all functions commented
-- [ ] Verify the unit tests are testing the code to perform the intended functionality
-- [ ] Verify the unit tests cover the positive and negative scenarios
-- [ ] Verify the screenshot tests are added and are they comprehensive
-- [ ] Verify documentation (design/api/example) has been added/updated
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+
+<!--- Describe your changes in detail -->
+
+## Motivation and Context
+
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## How has this been tested?
+
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, tests ran to see how -->
+<!--- your change affects other areas of the code, etc. -->
+
+## Screenshots (if appropriate):
+
+## Types of changes
+
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## Checklist:
```

Comment:
I think this list can be removed. It will also not be filled and it is not complete.